yang-packages/draft-ietf-netmod-yang-packages.txt

```diff
@@ -1159,7 +1146,7 @@ Internet-Draft                YANG Packages                February 2022
      // and remove this note.
      // RFC Ed.: replace XXXX with actual RFC number and remove this
      // note.
-     revision 2021-10-25 {
+     revision 2022-02-08 {
        rev:revision-label 0.2.0;
```

Comment:
Unrelated to this change: the revision-label is supposed to have the draft ID.

Reply:
I had fixed this to 0.3.0-draft-ietf-netmod-yang-packages-03; in my previous PR. Maybe just use the same for all 4 other modules.